UDPClientGUI/MyUDPClient.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPCommGUI
+{
+    class MyUDPClient : IClientInterface
+    {
+
+        private UdpClient client;
+    
+        public MyUDPClient()
+        {
+            this.ConnectionStatus = false;
+        }
+
+        public string ConnectionIP { get; set; }
+
+        public int ConnectionPort { get; set; }
+
+        public string InputMessage{ get; set; }
+
+        public bool ConnectionStatus { get; set; }
+                
+        public void Connect()
+        {
+            this.client = new UdpClient();
+            client.Connect(ConnectionIP, ConnectionPort);
+            this.ConnectionStatus = true;
+        }
+
+        public void Disconnect()
+        {
+            if(ConnectionStatus)
+            {
+                client.Close();
```

Comment:
It's a good idea to Dispose of the UDP client when the application closes if you haven't already.

Reply:
Implemented for both server and client